js/languages/en-US.json

```diff
@@ -833,6 +838,34 @@
       "emptyData": "No valid data was returned from %{provider}."
     }
   },
+  "onboarding": {
+    "pageOfPage": "%{startPage} of %{totalPages}",
+    "profileFailedSaveTitle": "Unable To Save Your Profile",
+    "settingsFailedSaveTitle": "Unable To Save Your Settings",
+    "settingsFailedSaveAvatar": "Unable To Save Your Avatar",
+    "introScreen": {
+      "connectionLbl": "Connection:",
+      "btnChange": "Change",
+      "introLine": "A free marketplace. No fees. No restrictions. Earn Bitcoin",
```

Comment:
I was just matching the design on that one. If the design change, I imagine @morebrownies could polish this up.